File: Blanketi-WEB/April24/Frontend/app/Projekcija.js
export class Projekcija{

    constructor(id,naziv,vremePrikazivanja,sifra,sala,karte = []) {
        this.id = id;
        this.naziv = naziv;
        this.vremePrikazivanja = vremePrikazivanja;
        this.sifra = sifra;
        this.sala = sala;
        this.karte = karte;
    }
}

File: Blanketi-WEB/April24/Backend/Models/Projekcija.cs
namespace WebTemplate.Models;

public class Projekcija
{
    [Key]
    public int ID { get; set; }

    [MaxLength(70)]
    public required string Naziv { get; set; }

    public required DateTime VremePrikazivanja { get; set; }

    public Sala? sala { get; set; }

    public required string Sifra { get; set; }

    public required uint OsnovnaCenaKarte { get; set; }

    public List<Karta>? Karte { get; set; }


}